dcs/urls.py

```diff
@@ -10,9 +10,11 @@
     # url(r'^blog/', include('blog.urls')),
 
     url(r'^$', 'dcs.views.home', name='home'), # mock, CHANGE later!
+    url(r'^nimda/', include('grappelli.urls')),
     url(r'^admin/', include(admin.site.urls)),
     url(r'^accounts/', include('registration.backends.default.urls')),
     url(r'^', include('multimedia.urls', namespace='multimedia')),
+    url(r'^carts/', include('transaction.urls')),
```

Comment:
I think this line needs to come before the multimedia url routing, but I'm not sure. But regardless, don't forget to namespace it later